mis_builder/models/aep.py

```diff
@@ -238,7 +241,7 @@ def get_aml_domain_for_dates(self, date_from, date_to,
         elif mode == self.MODE_UNALLOCATED:
             date_from_date = fields.Date.from_string(date_from)
             fy_date_from = \
-                self.company.\
+                self.company_ids[0].\
```

Comment:
Hm. Here it's a special treatment for the first company. This could lead to surprises. 

Could you extract the `fy_date_from` computation in a separate method and add a check (raise UserError) that all companies have the same `fy_date_from`? Later we can add logic to handle companies with different fiscal years.